packages/atomic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -275,7 +275,7 @@ describe('Facet v1 Test Suites', () => {
       });
 
       describe('when searching for a value that returns a single result', () => {
-        const query = 'Gregory';
+        const query = 'BillV';
```

Comment:
I would mock the response to remove all results besides the first one.

Reply:
I prefer to keep using the actual API (not a mock) unless absolutely necessary.

It does makes our tests a bit more finicky, but we've also caught quite a bit of actual API production issue by not mocking stuff.